lib/bolt/core/src/dep/terraform/gen.rs

```diff
@@ -483,6 +483,43 @@ async fn vars(ctx: &ProjectContext) {
 		vars.insert("s3_providers".into(), s3_providers(ctx).await.unwrap());
 	}
 
+	// BetterUptime
+	if let Some(betteruptime) = &config.betteruptime {
+		// Make sure there is at least one pool
+		// if config.pools.is_empty() {
+		// 	panic!("BetterUptime requires at least one pool, otherwise it will not be able to monitor the service");
+		// }
+
+		// Load all the regions of pools
+		let regions = &config
+			.pools
+			.iter()
+			.filter_map(|pool| match config.regions.get(&pool.region) {
+				Some(region) => Some((pool.region.clone(), region.provider_region.clone())),
+				None => None,
+			})
+			.collect::<HashSet<_>>();
+
+		// let public_ip = ctx.domain_main_api().unwrap();
```

Comment:
@NathanFlurry @MasterPtato I have this commented out for testing, but is this the proper way to get the api on production clusters?

Reply:
Almost. I would recommend using `ProjectContextData::origin_api` since it includes the scheme in it. For example, if exposing a raw IP, (i.e. `http://1.2.3.4:8080` vs `https://api.rivet.gg`).

Reply:
Resolved in https://github.com/rivet-gg/rivet/commit/cd82d096327f98379148e59415b484eb649ba14c